ProcessLib/HydroMechanics/HydroMechanicsProcessData.h

```diff
@@ -104,8 +105,19 @@ struct HydroMechanicsProcessData
     double dt = 0.0;
     double t = 0.0;
 
+    /// Fluid's compressibility. A scalar quantity,
+    /// ParameterLib::Parameter<double>.
+    double const fluid_compressibility;
+
+    /// Reference Temperature. A scalar quantity.
     double const reference_temperature;
 
+    /// Specific gas constant. A scalar quantity.
+    double const specific_gas_constant;
+
+    /// Fluid type. 0 = incompressible_fluid, 1 = ideal_gas
```

Comment:
Did you forget to mention the compressible_fluid type (somewhere it is handled in the code) to mention in the docu or is there a reason that it is not documented here?